server/lib/queries.js

```diff
@@ -1125,6 +1125,38 @@ const getTaxFormsRequiredForAccounts = async (accountIds = [], year) => {
   return getTaxFormsOverTheLimit(results, 'collectiveId');
 };
 
+/**
+ * Returns the contribution or expense amounts over time.
+ */
+const getTransactionsTimeSeries = async (kind, type, hostCollectiveId, timeUnit, collectiveIds, dateFrom, dateTo) => {
+  return sequelize.query(
+    `SELECT DATE_TRUNC(:timeUnit, "createdAt") AS "date", sum(amount) as "amount", "currency"
```

Comment:
Better look at the amountInHostCurrency and group by hostCurrency.

Reply:
Corrected this one to include the `amountInHostCurrency` and `hostCurrency`.